boards/arm/efm32gg_stk3701a/board.h

```diff
@@ -9,18 +9,18 @@
 #define __INC_BOARD_H
 
 /* This pin is used to enable the serial port using the board controller */
-#define BC_ENABLE_GPIO_NAME		DT_GPIO_GECKO_PORTE_NAME
+#define BC_ENABLE_GPIO_NAME		"GPIO_E"
```

Comment:
This should come from DTS. However, I would prefer to fix it in subsequent PR as it requires adding a new binding.